docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,37 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
+Documentation: https://chanzuckerberg.github.io/cellxgene-census/
+Contact: cellxgene@chanzuckerberg.com
+ManagedBy: "[Chan Zuckerberg Initiative](http://www.chanzuckerberg.com/)"
+UpdateFrequency: New releases are published weekly. LTS releases are published every 6 months.
+Tags:
+  - single-cell
+  - genomics
+  - computational biology
+License: MIT License
```

Comment:
nit: Since this is the license for the Census data, should this be CC-BY license?

Source: https://github.com/chanzuckerberg/cellxgene-census#reuse

Reply:
agree - pretty sure MIT is wrong license

Reply:
Correct. I'll modify it accordingly.